Gems/Blast/Code/Source/Family/DamageManager.h

```diff
@@ -10,6 +10,7 @@
 #include <Blast/BlastMaterial.h>
 #include <Family/ActorTracker.h>
 #include <Blast/BlastSystemBus.h>
+#include <AZCore/Interface/Interface.h>
```

Comment:
The include path has a case-sensitive typo that prevent this from compiling on case-sensitive platforms like Linux

```suggestion
#include <AzCore/Interface/Interface.h>
```